example-apps/token_curated_registry/db/mapper.go

```diff
@@ -0,0 +1,161 @@
+package db
+
+import (
+	"github.com/cosmos/cosmos-academy/example-apps/token_curated_registry/types"
```

Comment:
Call it instead: 
tcr "github.com/cosmos/cosmos-academy/example-apps/token_curated_registry/types"